frontend/packages/dev-console/src/components/import/advanced/ServerlessScalingSection.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import { NumberSpinnerField } from '../../formik-fields';
+import FormSection from '../section/FormSection';
+
+const ServerlessScalingSection: React.FC = () => {
+  return (
+    <FormSection title="Scaling">
```

Comment:
Duplicate scaling section?

Reply:
need to check existing scaling section and if we can make use of same or extend

Reply:
yes had a separate component for knative scaling as it's different than normal scaling form. Even `subTitle` is not there for serverless scaling. Have put a check in to load appropriate component thus  would be part of the same `Scale` section rather than a section called `Serverless Scale`

```
<ProgressiveListItem name="Scale">
        {values.serverless.enabled ? <ServerlessScalingSection /> : <ScalingSection />}
      </ProgressiveListItem>
```

Reply:
maybe change the `scale` section to use `scale` and then you can use `scaling` or `serverlessScaling` here.

Reply:
@joshuawilson have handled it , both `<ScalingSection />` and `<ServerlessScalingSection />` are under same section `scale`

Reply:
My question was more to do with IF we should have 2 sections or just 1. How much is shared? Is it just a similar name? @christianvogt

Reply:
To be clear, I'm addressing the link in the advanced line at the bottom of the form.

Reply:
@joshuawilson both `ServerlessScalingSection` and `ScalingSection` are quite different IMO should have it different. Scaling has replicas just one field and we have  like four different fields with different default and validations